packages/mask/package.json

```diff
@@ -123,7 +123,12 @@
     "webcrypto-liner": "1.2.4",
     "webextension-polyfill": "^0.8.0",
     "z-schema": "^5.0.2",
-    "zod": "3.11.6"
+    "zod": "3.11.6",
+    "react-icons": "4.3.1",
```

Comment:
We already have 2 icon packs. Please use either `@mui/icons-material` or `react-feather`.